### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by toddmotto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+## Overview
+
+Now that we know what transclusion does, we can look into more advanced methods of implementing it. Let's take a look how to transclude our content manually.
+
+## Objectives
+
+- Use the transclude function inside the link function
+
+## transclude()
+
```
Comment (ipc103) [inline README.md:12]: Can you set this up by explaining why we might need this? What's an example of a situation where you'd need to use manual transclusion?

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+    }
+  };
+}
+
+angular
+  .module('app', [])
+  .directive('ourDirective', ourDirective);
+```
+
+Now that we've got the transcluded elements as actual DOM elements and also our whole directive element (the `element` variable), we can insert our transcluded content wherever we want. If we want to put it into the span, we can do this:
```
Comment (ipc103) [inline README.md:37]: Can you show us an example of what will the `transclude` argument will look like?

**Comment by ipc103** [comment]
Another nice demo of the implementation. Can you explain a little bit at the beginning why you might choose this over the style we saw in the previous README, and how the students can tell the difference?

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+function ourDirective() {
+  return {
+    transclude: 'element',
+    template: [
+      '<div class="ourDirective">',
+        'The content of our directive is: <span></span>',
+      '</div>'
+    ].join(''),
+    link: function (scope, element, attrs, ctrl, transclude) {
+        el.find('span').after(transclude());
```
Comment (ipc103) [inline README.md]: i'm texting this out in the context of the next lab - what is `el` here?

File: README.md (hunk 2 above)
Comment (toddmotto) [inline README.md:37]: Not sure what you mean here?

**Pull request merged**